tests/test_sync_canary.py

```diff
@@ -0,0 +1,23 @@
+from tap_tester import connections
+from base import TestLinkedinAdsBase
+
+class LinkedinAdsSyncTest(TestLinkedinAdsBase):
+
+    @staticmethod
+    def name():
+        return "tap_tester_linkedin_ads_sync_test"
+
+    def test_run(self):
+        """
+        Testing that sync creates the appropriate catalog with valid metadata.
+        Verify that all fields and all streams have selected set to True in the metadata
+        """
+        conn_id = connections.ensure_connection(self)
+
+        found_catalogs = self.run_and_verify_check_mode(conn_id)
+
+        self.perform_and_verify_table_and_field_selection(conn_id,found_catalogs)
+
+        record_count_by_stream = self.run_and_verify_sync(conn_id)
+
+        self.assertGreater(sum(record_count_by_stream.values()), 0)
```

Comment:
For the canary test, we want to verify that all expected streams are replicating data, so same assertion but in a for loop iterating over the streams.

Reply:
Done